package.json

```diff
@@ -121,15 +121,21 @@
   "eslintConfig": {
     "extends": [
       "airbnb-base",
```

Comment:
I ran into numerous eslint errors while writing the typescript for this and did a little research, it seems we needed to extend our base eslint config with this plugin. You'll need to re-up your dependencies locally.